src/main/kotlin/dynamic_programming/PalindromePartitioning.kt

```diff
@@ -0,0 +1,75 @@
+package dynamic_programming
+
+/**
+ * Palindrome Partitioning Algorithm
+ *
+ * You are given a string as input, and task is to find the minimum number of partitions to be made,
+ * in the string sot that the resulting strings are all palindrome
+ *
+ * Time Complexity = O(n^2)
+ *
+ * */
+
+
+/**
+ * @param String is the string to be checked
+ * @param Int is the starting index of the string in consideration
+ * @param Int is the ending index of the string in consideration
+ * @return whether string is a palindrome or not
+ **/
+fun isPalindrome(string: String, i: Int, j: Int): Boolean {
+    for (l in 0..(j - i) / 2) {
+        if (string[l + i] != string[j - l]) {
+            return false
+        }
+    }
+    return true
+}
+
+
+/**
+ * @param String is the string to be checked
+ * @param Int is the starting index of the string in consideration
+ * @param Int is the ending index of the string in consideration
+ * @return minimum number of partitions required
+ **/
+fun palindromePartition(string: String, i: Int, j: Int): Int {
```

Comment:
give meaningful names to variables

Reply:
I have used generic names for indices(i and j), and given a proper description of them, I have done this in order to keep the variable names.